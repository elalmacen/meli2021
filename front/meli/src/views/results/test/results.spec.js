import React from 'react';
import {shallow} from 'enzyme';
import View from '../results';

describe('test searchBox component', () => {
    test('renders without crashing', () => {
        const wrapper = shallow(<View searchParam={'batman'}/>);
        expect(wrapper).toMatchSnapshot();
    });    
});
