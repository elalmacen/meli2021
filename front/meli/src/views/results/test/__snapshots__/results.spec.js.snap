// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test searchBox component renders without crashing 1`] = `
<Fragment>
  <ProductList
    categories={Array []}
    products={Array []}
  />
</Fragment>
`;
